app/livechat/client/views/app/tabbar/visitorInfo.js

```diff
@@ -93,6 +93,10 @@ Template.visitorInfo.helpers({
 
 	customVisitorFields() {
 		const customFields = Template.instance().customFields.get();
+		const canViewCustomFields = hasAtLeastOnePermission(['view-livechat-room-customfields', 'edit-livechat-room-customfields']);
```

Comment:
The `canViewCustomFields` const is used just to check the permission, why not do something like this?
```suggestion
		if (!hasAtLeastOnePermission(['view-livechat-room-customfields', 'edit-livechat-room-customfields'])) {
			return;
		}
```